scss/_nav.scss

```diff
@@ -213,8 +213,8 @@
 .nav-justified {
   > .nav-link,
   .nav-item {
-    flex-basis: 0;
     flex-grow: 1;
+    flex-basis: 0;
```

Comment:
This is not yet modified in Bootstrap (upstream), but this is needed to fix the lint issue here. It depends on a complexity around sass/stylelint/etc. and the precise content of `package-lock.json`. Since it has no impact whatsoever, I've modified it anyway here.